src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('LocalGameWrapperComponent', () => {
     let componentTestUtils: ComponentTestUtils<P4Component>;
     const O: Player = Player.ZERO;
     const X: Player = Player.ONE;
-    const _: Player = Player.NONE;
+    const _: PlayerOrNone = Player.NONE;
```

Comment:
what the hell :D ?

Reply:
plus clair maintenant ?

Reply:
Oui mais du coup p-e mettre tout (X, O, _) en tant que PlayerOrNone, c'est plus sexy non ?

Reply:
je dirais non, si t'as besoin d'un Player, d'un coup tu va devoir caster tes X/O, alors que tu *sais* que ce sont des Player, pas fou ?

Reply:
Ouais je veux dire, si tu as un Table<PlayerOrNone> autant que les variable que tu met dedans le soient aussi, c'est le cas des X, O, _ !

Reply:
yep, c'est donc bien fait